Pod/Classes/VOKCoreDataManager.m

```diff
@@ -112,13 +127,23 @@ - (NSPersistentStoreCoordinator *)persistentStoreCoordinator
     return _persistentStoreCoordinator;
 }
 
+- (NSBundle *)bundleForModel
+{
+    if (!_bundleForModel) {
+        //Default to using the bundle for this class.
+        _bundleForModel = [NSBundle bundleForClass:[self class]];
+    }
+    
+    return _bundleForModel;
```

Comment:
Not easier... not easy by any means.  But if `_bundleForModel` is `nil`, then `- bundleForModel` will return the default value, whereas if `_bundleForNil` is non-`nil`, it'll return whatever its value is.  Whereas your code will lose the information that it was intended to be the default value.  :tomato: :tomato: (It was more of a thing when I was thinking about it being null-resettable, but that's already the case with your code, so :shrugs:.)

You're right that it's more backward-compatible in terms of not changing behavior of existing code and that's probably enough reason to leave it alone.  I can't see why making it the main bundle would be a problem, but :shrugs:.